### ISSUE TITLE
In tq_mutate col_rename doesn't accept renaming to something that includes letters low for example ma_slow

### ISSUE THREAD

**Issue by rjvelasquezm** [business-science/tidyquant#137]
In the replace_bad_names function seems its setup to look for 'low' as part of an OHLC format, but it also detects titles with those three letters, for example 'ma_slow' for naming a slow moving average. Suggestion to change the regex so that it only selects specifically low.

**Comment by mdancho84** [comment]
Yes, I think the function needs to be restructured. It's attempting to prevent collision with OHLCV column names.
